test/e2e/main.go

```diff
@@ -0,0 +1,73 @@
+package main
```

Comment:
I'd scope this to `test/openshift/e2e` @frobware @ingvagabund wdyt?

Reply:
sgtm